spring-rabbit/src/main/java/org/springframework/amqp/rabbit/annotation/RabbitListener.java

```diff
@@ -332,4 +332,18 @@
 	 */
 	String converterWinsContentType() default "true";
 
+	/**
+	 * Override the container factory's {@code batchListener} property. The listener
+	 * method signature should receive a {@code List<?>}; refer to the reference
+	 * documentation. This allows a single container factory to be used for both record
+	 * and batch listeners; previously separate container factories were required.
+	 * @return "true" for the annotated method to be a batch listener or "false" for a
+	 * single message listener. If not set, the container factory setting is used. SpEL and
+	 * property place holders are not supported because the listener type cannot be
```

Comment:
So, if they are not supported, then why the property is `String` but not `boolean`?
Or the idea is because there is no way to have it set to something like `null`, so we can rely on the container factory option?
Then maybe `ValueConstants.DEFAULT_NONE` would be better?

Reply:
Well, if we change this one, we should change the others; I made a quick attempt at that and it is not trivial. I prefer to leave this as-is, at least for now.